lib/assessment.js

```diff
@@ -20,7 +24,10 @@ var sum = function sum() {
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  var length = this.length;
+
 
+return newArray;
 };
```

Comment:
Since `newArray` is the variable this function is assigned to, and also the name for the function within itself, this function returns itself. Functions that return themselves are, in most cases, not what you want to create.